mrtt-ui/src/components/CheckboxGroup/CheckboxGroup.stories.js

```diff
@@ -39,3 +39,14 @@ export const OtherOptionPrePopulated = () => (
     shouldAddOtherOptionWithClarification={true}
   />
 )
+
+export const WithNestedMarkupForSelected = () => (
+  <CheckboxGroup
+    options={options}
+    value={{ selectedValues: ['Option 1', 'Option 2'], otherValue: 'Some other value' }}
+    onChange={action('onChange')}
+    id='cuddlyKittens'
```

Comment:
cute id name but why is this here 🐱?

Reply:
we need an id to create a unique optionId for each option and the label for each option vs the group of checkboxes as a whole to make them accessible. Otherwise with no id, if we have two checkbox groups on a single page, we get non-unique option ids and the htmlFors get weird.

Reply:
ok, can we rename it to `option` or something specific to it?

Reply:
I like using id. People know it needs to be unique, and understand what it is right away. Since its used to generate other unique ids, it fits.

Reply:
using id is fine, I just mean can we rename something other than `cuddlyKittens` @mmnoo
<img width="244" alt="Screen Shot 2022-06-17 at 12 56 13 PM" src="https://user-images.githubusercontent.com/26089140/174343553-5a98fafa-0c1b-47c2-a412-2b03cab21f85.png">

Reply:
We could, but its a bit too much of a nitpick IMO. This is a story for demo, not code where the meaning of the id needs to be descriptive and void of personality. Its totally ok to use random silly values for mock data that only get used in tests for example (as long as its not offensive, racist, mean spirited, etc.) 

That is to say, I get the rule you are referring to, I just dont think it is appropriately applied here.

Reply:
I missed that this was in stories earlier! sorry about that - that's fine with me 👍